src/usecase/group/groupUsecase.ts

```diff
@@ -207,7 +243,7 @@ export class GroupUsecase {
         throw new Error("Cannot kick the group owner");
       }
 
-      await prisma.$transaction(async (tx) => {
+      await prisma.$transaction(async () => {
```

Comment:
wrong transaction handling
need to be:
```suggestion
      await prisma.$transaction(async (tx) => {
        // then we pass this tx to a repo method
        await this.groupRepository.removeUserFromGroup(memberUserId, ownerGroup.id, tx) {

```